crates/icondiffbot2/src/main.rs

```diff
@@ -95,14 +94,21 @@ fn read_key(path: &Path) -> Vec<u8> {
     key
 }
 
-#[tokio::main]
+const JOB_JOURNAL_LOCATION: &str = "jobs";
+
+#[actix_web::main]
 async fn main() -> std::io::Result<()> {
+    std::env::set_current_dir(std::env::current_exe().unwrap().parent().unwrap()).unwrap();
```

Comment:
Iiii dunno if this is a good idea... This seems like something that people running it should figure out.. this would prevent you from, say, setting this up as a systemd service with a job journal in a common location

Reply:
they shouldn't be, or weird things will happen

Reply:
oh? what kind of weird things?

Reply:
the app changes cwd regularly, and yaque will fail to delete the lock file if it can't find the job journal with relative paths

Reply:
this cwd-based pathing is awful, ill probably redo it sometime later

Reply:
redone